shared/oae/css/oae.base.css

```diff
@@ -904,24 +904,27 @@ ul.as-list li.as-result-item.active {
 
 .oae-markdown-embedded table {
     border-bottom: 1px solid #ECECEC;
-    font-size: 14px;
     margin: 20px 0;
 }
 
 .oae-markdown-embedded table th {
     background-color: #FFF;
     border-bottom: 1px solid #999;
     border-top: 1px solid #ECECEC;
-    font-weight: bolder;
+    font-weight: 500;
```

Comment:
500 is not showing as bold to me. This should be 600 or 700 to match the style in the designs.